manim/scene/scene.py

```diff
@@ -503,10 +518,50 @@ def remove(self, *mobjects):
                 self.restructure_mobjects(mobjects, list_name, False)
             return self
 
-    def add_updater(self, func):
+    def add_updater(self, func: Callable[[float], None]) -> None:
+        """Add an update function to the scene.
+
+        The scene updater functions are run every frame,
+        and they are the last type of updaters to run.
+
+        .. WARNING::
+
+            When using the Cairo renderer, scene updaters that
+            modify mobjects are not detected in the same way
+            that mobject updaters are. To be more concrete,
+            a mobject only modified via a scene updater will
+            not necessarily be added to the list of *moving
+            mobjects* and thus might not be updated every frame.
+
+            TL;DR: Use mobject updaters to update mobjects.
```

Comment:
Minor comment: The [numpy style guide](https://numpydoc.readthedocs.io/en/latest/format.html#warnings) suggests we include a warning after `Parameters` and before 
`see also`. But I can also see how it is useful to include it after the description as well.

Reply:
I see this warning more as a part of the extended summary than a separate section, so I think it would still be fine. Thanks for the review!

Reply:
I agree, it is appropriate as it is.
Also welcome! :)